keyboards/1upkeyboards/1upsuper16v3/info.json

```
@@ -0,0 +1,195 @@
+
+        { "flags": 2, "x": 4, "y": 52 },
+        { "flags": 2, "x": 4, "y": 60 },
+        { "flags": 2, "x": 14, "y": 63 },
+
+        { "flags": 4, "matrix": [0, 12], "x": 28, "y": 56 },
+
+        { "flags": 2, "x": 42, "y": 63 },
+        { "flags": 2, "x": 70, "y": 63 },
+
+        { "flags": 4, "matrix": [0, 13], "x": 84, "y": 56 },
+
+        { "flags": 2, "x": 99, "y": 63 },
+        { "flags": 2, "x": 126, "y": 63 },
+
+        { "flags": 4, "matrix": [0, 14], "x": 140, "y": 56 },
+
+        { "flags": 2, "x": 154, "y": 63 },
+        { "flags": 2, "x": 182, "y": 63 },
+
+        { "flags": 4, "matrix": [0, 15], "x": 196, "y": 56 },
+
+        { "flags": 2, "x": 210, "y": 63 },
+        { "flags": 2, "x": 220, "y": 60 },
+        { "flags": 2, "x": 220, "y": 52 },
+        { "flags": 2, "x": 220, "y": 44 },
+
+        { "flags": 4, "matrix": [0, 11], "x": 196, "y": 40 },
+        { "flags": 4, "matrix": [0, 10], "x": 140, "y": 40 },
+
+        { "flags": 2, "x": 220, "y": 36 },
+        { "flags": 2, "x": 220, "y": 28 },
+
+        { "flags": 4, "matrix": [0, 7], "x": 196, "y": 24 },
+        { "flags": 4, "matrix": [0, 6], "x": 140, "y": 24 },
+
+        { "flags": 2, "x": 220, "y": 20 },
+        { "flags": 2, "x": 220, "y": 12 },
+
+        { "flags": 4, "matrix": [0, 3], "x": 196, "y": 8 },
+        { "flags": 4, "matrix": [0, 2], "x": 140, "y": 8 },
+
+        { "flags": 2, "x": 220, "y": 4 },
+        { "flags": 2, "x": 210, "y": 1 },
+        { "flags": 2, "x": 182, "y": 1 },
+        { "flags": 2, "x": 154, "y": 1 },
+        { "flags": 2, "x": 126, "y": 1 }
+
+        ]
+    },
+    "layouts": {
+        "LAYOUT_ortho_4x4": {
+            "layout": [
+                { "label": "00", "matrix": [0, 0], "w": 1, "x": 0, "y": 0 },
+                { "label": "01", "matrix": [0, 1], "w": 1, "x": 1, "y": 0 },
+                { "label": "02", "matrix": [0, 2], "w": 1, "x": 2, "y": 0 },
+                { "label": "03", "matrix": [0, 3], "w": 1, "x": 3, "y": 0 },
+                { "label": "10", "matrix": [0, 4], "w": 1, "x": 0, "y": 1 },
+                { "label": "11", "matrix": [0, 5], "w": 1, "x": 1, "y": 1 },
+                { "label": "12", "matrix": [0, 6], "w": 1, "x": 2, "y": 1 },
+                { "label": "13", "matrix": [0, 7], "w": 1, "x": 3, "y": 1 },
+                { "label": "20", "matrix": [0, 8], "w": 1, "x": 0, "y": 2 },
+                { "label": "21", "matrix": [0, 9], "w": 1, "x": 1, "y": 2 },
+                { "label": "22", "matrix": [0, 10], "w": 1, "x": 2, "y": 2 },
+                { "label": "23", "matrix": [0, 11], "w": 1, "x": 3, "y": 2 },
+                { "label": "30", "matrix": [0, 12], "w": 1, "x": 0, "y": 3 },
+                { "label": "31", "matrix": [0, 13], "w": 1, "x": 1, "y": 3 },
+                { "label": "32", "matrix": [0, 14], "w": 1, "x": 2, "y": 3 },
+                { "label": "33", "matrix": [0, 15], "w": 1, "x": 3, "y": 3 }
```

`w: 1` is default key width
```suggestion
                { "label": "00", "matrix": [0, 0], "x": 0, "y": 0 },
                { "label": "01", "matrix": [0, 1], "x": 1, "y": 0 },
                { "label": "02", "matrix": [0, 2], "x": 2, "y": 0 },
                { "label": "03", "matrix": [0, 3], "x": 3, "y": 0 },
                { "label": "10", "matrix": [0, 4], "x": 0, "y": 1 },
                { "label": "11", "matrix": [0, 5], "x": 1, "y": 1 },
                { "label": "12", "matrix": [0, 6], "x": 2, "y": 1 },
                { "label": "13", "matrix": [0, 7], "x": 3, "y": 1 },
                { "label": "20", "matrix": [0, 8], "x": 0, "y": 2 },
                { "label": "21", "matrix": [0, 9], "x": 1, "y": 2 },
                { "label": "22", "matrix": [0, 10], "x": 2, "y": 2 },
                { "label": "23", "matrix": [0, 11], "x": 3, "y": 2 },
                { "label": "30", "matrix": [0, 12], "x": 0, "y": 3 },
                { "label": "31", "matrix": [0, 13], "x": 1, "y": 3 },
                { "label": "32", "matrix": [0, 14], "x": 2, "y": 3 },
                { "label": "33", "matrix": [0, 15], "x": 3, "y": 3 }
```